msal/src/Microsoft.Identity.Client/Exceptions/MsalClientException.cs

```diff
@@ -154,28 +154,17 @@ public class MsalClientException : MsalException
 #if ANDROID
 
         /// <summary>
-        /// Xamarin.Android specific. This error indicates that chrome, or a browser implementing chrome tabs, is not installed on the device. 
-        /// The library sdk uses chrome custom tab for authorize request if applicable or falls back to chrome browser.
-        /// <para>Mitigation</para>If you really need to use the System web browser (for instance to get SSO with the browser), notify the end 
-        /// user that chrome or a browser implementing chrome custom tabs needs to be installed on the device. 
-        /// Otherwise you can also use <see cref="UIParent.IsSystemWebviewAvailable"/> to check if a required browser is available on the device
-        /// and require the library to use the embedded web view if it is not by setting the boolean to <c>true</c> in the following
-        /// constructor: <see cref="UIParent.UIParent(Android.App.Activity, bool)"/>
-        /// <para>For more details</para> See https://aka.ms/msal-net-uses-web-browser
-        /// </summary>
-        public const string ChromeNotInstalledError = "chrome_not_installed";
```

Comment:
>ChromeNotInstalledError [](start = 28, length = 23)

if someone is using these.... perhaps those should stay or alternatively we should have information saying those are deprecated? then remove them later? (same for the one below)

Reply:
Yes, that's a breaking change in the public API. we cannot remove it. you might want to add and [Obsolete] attribute with the error property set to true. In case people use it today, they will know they need to use `AndroidActivityNotFound` instead.

Reply:
I added them back and added obsolete. 

---
In reply to: [227689910](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227689910) [](ancestors = 227689910)

Reply:
added them back and added obsolete

---
In reply to: [227649302](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227649302) [](ancestors = 227649302)